site/webpack.config.js

```diff
@@ -0,0 +1,68 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    //css entries
+    .addStyleEntry('app-css', ['./web/assets/scss/librecores.scss',
+        './web/assets/scss/bootstrap-librecores.scss',
+        './web/assets/css/bootstrap-social.css',
+        './web/assets/css/font-awesome.min.css',
+        './web/assets/css/librecores.css'])
```

Comment:
Format this in a way that makes it easy to see the list of file. And add a trailing comma to make the list easy to extend without merge conflicts or ugly diffs.

```
.addStyleEntry('app-css', [
  './web/assets/scss/librecores.scss',
  './web/assets/scss/bootstrap-librecores.scss',
  './web/assets/css/bootstrap-social.css',
  './web/assets/css/font-awesome.min.css',
])
```

(Equally apply this to the other code in this file.)